specs/sharding/p2p-interface.md

```diff
@@ -29,30 +31,39 @@ The adjustments and additions for Shards are outlined in this document.
 
 ## New containers
 
-### ShardBlob
+### ShardBlobBody
+
+```python
+class ShardBlobBody(Container):
```

Comment:
I lean toward moving these into the consensus spec, but I see the argument for not

Reply:
ehh. maybe not